ci-operator/step-registry/openshift/e2e/test/hypershift-qe/openshift-e2e-test-hypershift-qe-chain.yaml

```diff
@@ -2,6 +2,7 @@ chain:
   as: openshift-e2e-test-hypershift-qe
   steps:
   - ref: cucushift-hypershift-extended-health-check
+  - ref: cucushift-hypershift-extended-global-pull-secret
```

Comment:
why do you put the check step into the e2e test part? I suggest checking it in the pre since it is a feature check

Reply:
Done by updating it to the `pre` section in chain: [cucushift-installer-rehearse-azure-aks-hypershift-ephemeral-creds-provision](https://github.com/openshift/release/blob/master/ci-operator/step-registry/cucushift/installer/rehearse/azure/aks/hypershift/ephemeral-creds/provision/cucushift-installer-rehearse-azure-aks-hypershift-ephemeral-creds-provision-chain.yaml)